elements/nuxeo-document-actions/nuxeo-document-parent-inspector-button.js

```diff
@@ -0,0 +1,210 @@
+          background-color: var(--nx-button-secondary_-_background-color);
+          color: var(--nx-button-secondary_-_color);
+        }
+        paper-button.secondary:hover,
+        paper-button.secondary:focus {
+          border: var(--nx-button-secondary-hover_-_border);
+          color: var(--nx-button-secondary-hover_-_color);
+        }
+        .facets {
+          display: flex;
+          flex-wrap: wrap;
+          width: 645px;
+        }
+        .schemas {
+          display: flex;
+          flex-wrap: wrap;
+          width: 645px;
+        }
+        .show-items {
+          margin: 10px;
+          color: black;
+          width: 33%;
+          padding: 5px 40px;
+          border: 2px solid;
+          border-radius: 30px;
+          text-align: center;
+        }
+        .scrollable {
+          margin-top: 24px;
+          padding: 0 24px;
+          @apply --layout-scroll;
+          @apply --layout-flex;
+          @apply --layout-vertical;
+        }
+        .actions {
+          @apply --buttons-bar;
+          @apply --layout-horizontal;
+          @apply --layout-justified;
+        }
+      </style>
+
+      <nuxeo-connection id="nxcon" user="{{currentUser}}"></nuxeo-connection>
+
+      <template is="dom-if" if="[[_isAvailable(document)]]">
+        <div class="action" on-tap="_openDialog">
+          <paper-icon-button icon="[[icon]]" noink aria-labelledby="label"></paper-icon-button>
+          <nuxeo-tooltip>[[i18n('parentInspectorButton.tooltip')]]</nuxeo-tooltip>
+        </div>
+      </template>
+
+      <nuxeo-dialog id="dialog" with-backdrop modal>
+        <div class="container">
+          <div class="parentscroll">
+            <div class="scrollable">
+              <div>
+                <h1>Parent Inspector</h1>
+                <div>
+                  <p>This is the parent inspector. It displays information about the parent of this document.</p>
+                  <table class="table">
+                    <tr>
+                      <td>Title:</td>
+                      <td>[[document.contextParameters.firstAccessibleAncestor.title]]</td>
+                    </tr>
+                    <tr>
+                      <td>Path:</td>
+                      <td>[[document.contextParameters.firstAccessibleAncestor.path]]</td>
+                    </tr>
+                    <template is="dom-if" if="[[currentUser.isAdministrator]]">
+                      <tr>
+                        <td>UID:</td>
+                        <td>[[document.contextParameters.firstAccessibleAncestor.uid]]</td>
+                      </tr>
+                    </template>
+                  </table>
+                </div>
+              </div>
+              <div class="facetscontainer">
+                <h1>Facets:</h1>
+                <div class="facets">
+                  <template is="dom-repeat" items="[[facets]]" as="facets">
+                    <div class="show-items">[[facets]]</div>
```

Comment:
We need to update the property name here as well:
```suggestion
                  <template is="dom-repeat" items="[[_facets]]" as="facets">
                    <div class="show-items">[[_facets]]</div>
```